.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules
 *.orig
 .tmp
 npm-debug.log
+coverage
```

Comment:
can we do `reports/coverage` (excluding reports)?
anticipating other reports to come